.github/workflows/docker_update_flow_visibility.yml

```diff
@@ -0,0 +1,52 @@
+# Anyone with write permissions to the antrea-io/antrea Github repository can
+# trigger this workflow manually, but please check with a maintainer first. The
+# workflow will tag and push flow visibility related external images into Antrea
+# Docker hub.
+name: Manually update flow visibility Docker images
+
+on:
+  workflow_dispatch:
+    inputs:
+      ch-operator-tag:
+        description: 'Tag for clickhouse-operator and metrics-exporter Docker image'
+        required: true
+      grafana-tag:
+        description: 'Tag for grafana'
+        required: true
+      ch-server-tag:
+        description: 'Tag for clickhouse-server'
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Push antrea/flow-visibility-clickhouse-operator
+        uses: akhilerm/tag-push-action@v2.0.0
+        with:
+          src: docker.io/altinity/clickhouse-operator:${{ github.event.inputs.ch-operator-tag }}
+          dst: |
+            docker.io/antrea/flow-visibility-clickhouse-operator:${{ github.event.inputs.ch-operator-tag }}
+      - name: Push antrea/flow-visibility-metrics-exporter
+        uses: akhilerm/tag-push-action@v2.0.0
+        with:
+          src: docker.io/altinity/metrics-exporter:${{ github.event.inputs.ch-operator-tag }}
+          dst: |
+            docker.io/antrea/flow-visibility-metrics-exporter:${{ github.event.inputs.ch-operator-tag }}
+      - name: Push antrea/flow-visibility-grafana
+        uses: akhilerm/tag-push-action@v2.0.0
+        with:
+          src: docker.io/grafana/grafana:${{ github.event.inputs.ch-server-tag }}
```

Comment:
"ch-server-tag" -> "grafana-tag"?

Reply:
Good catch! Thanks, Anlan.

Reply:
sorry for separating my comments. just realized we might also want to add flow-visibility.yml here: https://github.com/antrea-io/antrea/blob/main/ci/check-manifest.sh
could you help add it?

Reply:
Sure, will add it soon.